docs/windows.md

```diff
@@ -0,0 +1,48 @@
+# Windows Installation
+
+## Overview
+
+The installer will install Grafana Agent into the default directory `C:\Program Files (x86)\Grafana Agent`, optionally you can select to enable the [Windows Exporter](https://github.com/grafana/windows_exporter) with all default options. 
+
+## Installation
+
+![](./assets/remote_options.png)
```

Comment:
Sure! Does mean we lose the ability to easily spin up one pointing somewhere via the silent install option but users can always have a post-install script for that.

Reply:
right, I think in general most users are going to need to modify the config file after install _anyway_ since the defaults don't collect very much. Not a huge loss, but we can always backpedal on this